group1/requirements/static/js/password_validator.js

```diff
@@ -0,0 +1,83 @@
+// function to validate password, matches 6 to 20 characters which contain at least one numeric digit, one uppercase, one lowercase letter, a inspiring message and a spell
+
+$(document).ready(function () {
+
+    // listens to the user input at password text box, handles event when key up.
+    $("#id_password1").keyup(function () {
+        validatePassword();
+    });
+
+    //listens to the the password confirmation input
+    $("#id_password2").keyup(function () {
+        matchPasswords();
+    });
+
+    // removes alerts classes if either of the text box is empty
+    $("input").blur(function () {
+        cleanUp();
+    });
+
+
+    // matches both password
+    function matchPasswords() {
+        var pwd1 = $("#id_password1").val();
+        var pwd2 = $("#id_password2").val();
+        console.log(pwd1 + " " + pwd2);
```

Comment:
Were you including this for testing purposes while working on this feature? We probably shouldn't be logging the passwords to he console, even if they've been hashed. :P